src/options/options.js

```diff
@@ -0,0 +1,39 @@
+const defaultOptions = {
+  isJsonVisible: false,
+  prettierOptions: {
+    arrowParens: "avoid",
+    bracketSpacing: true,
+    jsxBracketSameLine: false,
+    printWidth: 80,
+    semi: true,
+    singleQuote: false,
+    tabWidth: 2,
+    trailingComma: "none",
+    useTabs: false
+  }
+};
+
+export default defaultOptions;
+
+export function validateOptions(options) {
+  const validationMap = Object.entries(defaultOptions.prettierOptions).reduce(
+    (acc, [key, val]) => {
+      acc[key] = typeof val;
+      return acc;
+    },
+    {}
+  );
+  const errors = [];
+
+  for (const [key, val] of Object.entries(options)) {
+    if (!Object.hasOwnProperty.call(validationMap, key)) {
+      errors.push(`${key} is an invalid option`);
+    }
+
+    if (typeof val !== validationMap[key]) {
```

Comment:
All of the current options are all a consistent, single type. This might have to be updated in the future if we support any options that can be multiple types.

Reply:
Prettier doesn't seem to have any [options](https://prettier.io/docs/en/options.html) like this, so it should be fine.